src/Components/Popover/Popover.js

```diff
@@ -133,6 +138,17 @@ const Popover = props => {
     close();
   };
 
+  React.useEffect(() => {
```

Comment:
makes me sad this and line 152 didn't import useEffect like useState.

Reply:
following existing pattern.